proto/nibiru/perp/v2/genesis.proto

```diff
@@ -23,15 +24,20 @@ message GenesisState {
       [ (gogoproto.nullable) = false ];
 
   uint64 dnr_epoch = 6;
-
+ 
   message TraderVolume {
     string trader = 1;
     uint64 epoch = 2;
     string volume = 3 [
       (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
       (gogoproto.nullable) = false
     ];
+
   }
+  
+  // For testing purpose, we allow the collateral to be set at genesis
+  string collateral_creator = 11;
+  string collateral_subdenom = 12;
```

Comment:
Let's use a single string for this like `collateral_denom` so that it can be any valid denomination: native, IBC, or token factory.

Reply:
- Addressed in [0065ae0](https://github.com/NibiruChain/nibiru/pull/1656/commits/0065ae0b57123bdf6e92bf19a6202bc5b6a0ebf9)